lectures/11.rendering_2.md

```diff
@@ -57,259 +57,968 @@ We are recording, aren't we?
 
 ![kitty cameraman](http://www.catster.com/wp-content/uploads/2015/06/335f4392f011a80324e09f5ace0b3f57.jpg)
 
+
 --- NEXT SLIDE ---
 
 ## Review
 
 Why isn't ray tracing appropriate for real time rendering?
+* Really slow because each pixel needs tracing of many rays.

```

Comment:
1. this needs to be shown via animation
2. @Alekssasho add more questions from last time